src/App.vue

```diff
@@ -96,24 +96,24 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
   private draftFiling: IncorporationFilingIF
   private isDraft: boolean = false
 
-  private async created (): Promise<any> {
-    // Mock the nrNumber and Data
-    this.setNameRequestState({ nrNumber: 'NR7654459', entityType: 'BC', filingId: null })
-    this.setCurrentDate(this.dateToUsableString(new Date()))
-
-    try {
-      // Retrieve draft filing if it exists for the nrNumber specified
-      this.draftFiling = await this.fetchDraft()
-
-      // Parse the draft data into the store if it exists
-      this.draftFiling && this.parseDraft(this.draftFiling)
-
-      // Inform the router view we are resuming a draft and to update ui
-      this.isDraft = true
-    } catch (e) {
-      // TODO: Catch a flag from the api, if there is an error to be handled.
-    }
-  }
+  // private async created (): Promise<any> {
```

Comment:
Ok for debugging but please work with Cameron on a solution.

Reply:
This one still requires some work - I'll get with Cameron though and see what we can do to resolve it properly

Reply:
Commit with this code commented out, and created a new ticket to resolve this separately.

Reply:
Created entity #2958 for this task